packages/material-ui/src/Select/Select.test.js

```diff
@@ -762,9 +762,6 @@ describe('<Select />', () => {
 
       it('should throw if non array', function test() {
         if (!/jsdom/.test(window.navigator.userAgent)) {
-          // afterEach is not run since the only test in this block is skipped
-          // this is likely a bug in mocha
```

Comment:
It was a bug in `mocha`.

Was throwing in `test:karma`. Found actor by bisecting with `const unitContext = require.context('../packages/material-ui/src/', true, /\/[A-Z].*\.test\.js$/);` in https://github.com/mui-org/material-ui/blob/e049ad83633881af4bccee1290fd8ceb1c099587/test/karma.tests.js#L13 where `[A-Z]` was bisected until I skipped individual tests with `describe.skip` which revealed `Select.test`

`

Reply:
@dependabot squash and merge